cmd/server/config.go

```diff
@@ -66,5 +77,17 @@ func (c *config) Validate() error {
 		return errGenesisBlockRequired
 	}
 
+	if len(c.TokenWhiteList) != 0 {
+		for _, token := range c.TokenWhiteList {
+			if !ethcommon.IsHexAddress(token) {
+				return errInvalidTokenAddress
```

Comment:
Do we want to lookup the list using the contract client to make sure they are all defined?

Reply:
Could go either way, theoretically as long as it's a valid address we shouldn't care.  Probably moot, but they could want to track info on a contract that doesn't have a public Symbol/Name